XenModel/Messages.resx

```diff
@@ -2557,6 +2557,9 @@ Do you want to assign it to the schedule '{2}' instead?</value>
   <data name="CHECKING_SAFE_TO_UPGRADE" xml:space="preserve">
     <value>Checking partition layout</value>
   </data>
+  <data name="CHECKING_SAFE_TO_UPGRADE_HOST" xml:space="preserve">
+    <value>Host partition layout</value>

```

Comment:
Better to be "Host partition layout check", consistent with other parallel messages.